includes/discovery/functions.inc.php

```diff
@@ -197,10 +197,8 @@ function discover_device(&$device, $force_module = false)
 //end discover_device()
 
 // Discover sensors
-function discover_sensor(&$valid, $class, $device, $oid, $index, $type, $descr, $divisor = 1, $multiplier = 1, $low_limit = null, $low_warn_limit = null, $warn_limit = null, $high_limit = null, $current = null, $poller_type = 'snmp', $entPhysicalIndex = null, $entPhysicalIndex_measured = null, $user_func = null, $group = null, $rrd_type = 'GAUGE')
```

Comment:
I suppose you have a reason to keep `$unused` where it is instead of removing it alltogether ... In order to avoid a crash if a `discover_sensor` call is not updated to the new signature ?

Reply:
Yeah, I suspect a lot of custom (local os) will call that method with the old signature.

Eventually, we'll remove usage of that function in mainstream LibreNMS, so I'm not going to break them at this point :)

Reply:
I thought about making that change here, but wanted to take baby steps.